README.md

```diff
@@ -1,126 +1,201 @@
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-<br/>
-<details open="open">
-<summary>Table of Contents</summary>
+# Create, deploy and operate 🐍 Python AI Actions <br/> and 🤖 Automations anywhere.
 
-- [What is Robo?](#what-is-robo)
-- [Libraries](#libraries)
-- [CLI](#cli)
-- [Licence](#licence)
+Robocorp is the easiest way to extend the capabilities of AI agents, assistants and copilots with custom actions, written in Python. Create and deploy tools, skills, loaders and plugins that securely connects any AI Assistant platform to your data and applications.
 
-</details>
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./docs/include/robocorp-flow-dark.webp">
+  <img alt="Text changing depending on mode. Light: 'So light!' Dark: 'So dark!'" src="./docs/include/robocorp-flow-light.webp">
+</picture>
+
+Looking for a replacement to RPA? Head over to our [Enterprise Python Automation site](https://robocorp.com/docs/quickstart-guide) for more.
 
 ---
 
-> **Warning**<br/>
-> Robo CLI is not yet recommended for production use, while libraries are ready for prime time. Refer to Robocorp's production tooling for managing Python projects [here](https://robocorp.com/docs/quickstart-guide/python).
+# 🏃‍♂️ Quickstart
+
+Install Robocorp Action Server:
+
+```sh
+# On macOS
+brew install robocorp/tools/action-server
 
-## What is Robo?
+# On Linux or Windows
+pip install robocorp-action-server
+```
+
+Bootstrap a new project from a template. You’ll be prompted for a name of the project:
 
-Robo is both a CLI for managing your Python projects, and a set of libraries for automating your tasks. It handles your entire automation development lifecycle. Robo's key features are:
+```sh
+action-server new
+```
 
-- **Isolated environments.** Get started without installing any other tools, not even Python. Robo automatically creates fully isolated environments for projects, which can be reproduced anywhere else.
-- **Batteries included.** Robo ships with a broad set of libraries for automating common tasks, be it browser automation, document processing, or navigating desktop applications.
-- **Reduce boilerplate.** Turn your Python snippet into a runnable automation with minimal effort. Robo can be used to execute your code in other environments, and deployed in Control Room without any infrastructure work.
-- **Troubleshoot easily.** Robo automatically traces your automation task, and create an easy-to-read report of what happened. You can quickly see if something went wrong - and how to fix it.
+Navigate to the freshly created project folder and start the server:
 
-Robo is designed to function as a single tool that helps you more easily get things done. It is entirely compatible with the PyPI and Conda ecosystems, but also brings in a set of libraries designed from the ground up to be focused on automation.
+```sh
+cd my-project
```

Comment:
```suggestion
cd my-first-action-server
```

WDYT? To cement the term even more and gently hint at the fact that it's your first, but many will follow!